src/components/molecules/UiPhoneNumber/_internal/UiPhoneNumberPrefix/UiPhoneNumberPrefix.vue

```diff
@@ -0,0 +1,153 @@
+<template>
+  <UiDropdown
+    v-model="defaultCountryCode"
+    :text="prefixButtonText"
+    :items="prefixCodes"
+    class="ui-phone-number-prefix"
+    :popover-attrs="{ class: 'ui-phone-number-prefix__popover' }"
+  >
+    <template
+      #toggle="{
+        text,
+        isOpen,
+        toggleHandler,
+      }"
+    >
+      <UiPhoneNumberPrefixToggle
+        :is-open="isOpen"
+        @click="toggleHandler"
+      >
+        {{ text }}
+      </UiPhoneNumberPrefixToggle>
+    </template>
+    <div
```

Comment:
should be 
```
<template #default>
  <div
    role="radiogroup"
    class="ui-dropdown__items"
  >
  </div>
</template>
```

Reply:
> Q: What's the advantage of this one you proposed over the one implemented by @sonya0504? 🙁

It's easiest to understand what we do. And I suggest to use `<template #default>` always when you use named slots.